devicon.json

```diff
@@ -7827,6 +7827,28 @@
         "color": "#1697F6",
         "aliases": []
     },
+    {
+        "name": "vyper",
+        "altnames": ["vyperlang"],
+        "tags": [
+            "programming",
+            "language",
+            "python",
+            "blockchain",
+            "ethereum"
```

Comment:
[Dev Protocol](https://devprotocol.xyz) is a community that powered and helped Vyper grow a lot. Maybe we could add it here as well.
```suggestion
            "ethereum",
            "dev-protocol"
```

Reply:
I don't think `original-wordmark` or `plain-wordmark` are needed on this one, because this is just a heavy styled "V", not actually a word.
I'm making this choice based on example of Vala language, where the `original` is also a styled "V" and the `original-wordmark` is the styled word "Vala".

---
I'm ok with the change, but `dev-protocol` is a new tag and I think we should have more opinions on that.
This is similar to `smart-contracts` tag suggestion on [Clarity](#1393) issue.

Reply:
I've tried those ones now and got this:

|`original-wordmark`|`plain-wordmark`|
|:-:|:-:|
|![vyper-original-wordmark](https://user-images.githubusercontent.com/76992016/194279428-3cf3addd-e6e0-456a-ab00-19f8c5d6a2e2.svg)|![vyper-plain-wordmark copy](https://user-images.githubusercontent.com/76992016/194279433-611aff65-a15e-4ea9-90d2-30df6e2fb2ae.svg)|

What do you think?

Reply:
Looks good, but why is `plain-wordmark` smaller than `original-wordmark`?

Reply:
That happened because I put them inside a table and both are following the size of title width. However, your question made me see some tiny difference in size though, that I've already fixed.

Now side-by-side:
<div align="center">
  <img width="300" src="https://user-images.githubusercontent.com/76992016/194287841-813af2bd-e895-4195-aefc-a1b41515deac.svg"/>
  <img width="300" src="https://user-images.githubusercontent.com/76992016/194291575-fb10a302-238b-4f7a-b463-4d1a48ee5e72.svg"/>
</div>

Reply:
Please add them to the PR :)

Reply:
It's approved from me now ✔️
@Panquesito7 @lunatic-fox 
Time to decide whether or not you want to add that tag.

@Panquesito7 
I'll remind you of [my comment](https://github.com/devicons/devicon/pull/1401#discussion_r988679050) from earlier too:
> Could you tell us some more about how they helped Vyper though? Looks like it's just a tool that the Vyper team has used to decentralize their funding, if that's the case I don't think we should add it as a tag

Reply:
Since it's no problem, I'm fine with adding the tag. I'm just waiting for @Panquesito7's answer too.